documentation/writing-policies-autogen.md

```diff
@@ -8,7 +8,7 @@ Kyverno solves this issue by supporting automatic generation of policy rules for
 
 This auto-generation behavior is controlled by the `pod-policies.kyverno.io/autogen-controllers` annotation. 
 
-By default, Kyverno inserts an annotation `pod-policies.kyverno.io/autogen-controllers=all`, to generate an additional rule that is applied to pod controllers: DaemonSet, Deployment, Job, StatefulSet.
+By default, Kyverno inserts an annotation `pod-policies.kyverno.io/autogen-controllers=all`, to generate an additional rule that is applied to pod controllers: DaemonSet, Deployment, Job, StatefulSet. This feature **DOES NOT** support DENY(Validate) policy and Generate policy.
```

Comment:
Maybe add this as a separate line so its obvious:

**Note**: The Auto-Gen feature is only supported for validation rules with patterns. Validate - Deny rules and Generate rules are not supported.